AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -22,10 +22,37 @@
 
 - (instancetype)init;
 
+/**
+ *  Set child methods
```

Comment:
These (and a few others) are really great comments.  Can you look at the Apple Doc format and make sure these will show up in the documentation when we regenerate it?  @nguyenhuy, could you regenerate the docs so we have an updated set before NSSpain?